tasks/main.yml

```diff
@@ -1,16 +1,24 @@
 ---
 # tasks file for molecule-validate-glance-deploy
 
+- name: Clean old openstack-ansible-ops dir if previously existing
+  file:
+    state: absent
```

Comment:
Why is it necessary to delete the repo? The git module used below will update the repo by default if it already exists.

Ref: https://docs.ansible.com/ansible/latest/modules/git_module.html#parameters

Reply:
I've seen some weird errors happening if the folder is there, for example the ansible task of cloning the repo will fail until the dir is manually removed